HW_4/js.js

```diff
@@ -0,0 +1,74 @@
+//////////////////////////////////// Task 1
+console.log("-------------------------------" + " Task 1");
+function numberIntoObject(val) {
+    if (isNaN(val)) {
+        console.log("It's not a number.")
+        return {};
+    } else if (val > 999) {
+        console.log("A too big number.");
+        return {};
+    } else if (val != Math.floor(val)) {
+        val = Math.floor(val);
+        console.log("You've entered not an integer. The value has been changed into " + val);
+    }
+
+    var obj = {
+        "units": 0,
```

Comment:
Здесь кавычки необязательны
```suggestion
        units: 0,
```